src/libstd/rt/unwind/gcc.rs

```diff
@@ -99,8 +92,6 @@ pub mod eabi {
     }
 
     #[lang="eh_personality"]
-    #[no_mangle] // referenced from rust_try.ll
-    #[allow(private_no_mangle_fns)]
```

Comment:
Does this pass all the tests? I vaguely remember seeing the `rust_eh_personality` symbol in a list of reachable symbols which was necessary for LTO working, but I may be remembering incorrectly.